### ISSUE TITLE
Pressing Terminate button gives walkback

### ISSUE THREAD

**Issue by ericwinger** [ericwinger/Jade#82]
- Press the Terminate button above the stack frame, get this walkback: 
- Confirm Yes to ``Terminate process?``

```
11:41:16 AM, Monday, August 20, 2018: 'RowanProcessService does not understand #terminate'
RowanProcessService(Object)>>doesNotUnderstand:
[] in RowanDebugger(JadeDebugger)>>_terminateProcess
ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry
[] in ExceptionHandler(ExceptionHandlerAbstract)>>try:
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
ExceptionHandler(ExceptionHandlerAbstract)>>try:
BlockClosure>>on:do:
RowanDebugger(JadeDebugger)>>_terminateProcess
RowanDebugger(JadeDebugger)>>terminateProcess
Symbol>>forwardTo:
CommandDescription>>performAgainst:
[] in Command>>value
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
Command>>value
ShellView>>performCommand:
RowanDebugger(Shell)>>performCommand:
CommandQuery>>perform
DelegatingCommandPolicy(CommandPolicy)>>route:
[] in PushButton(View)>>onCommand:
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
Cursor>>showWhile:
PushButton(View)>>onCommand:
PushButton(CommandButton)>>onActionPerformed
PushButton(View)>>performAction
PushButton>>command:id:
ContainerView(View)>>wmCommand:wParam:lParam:
ContainerView(View)>>dispatchMessage:wParam:lParam:
[] in InputState>>wndProc:message:wParam:lParam:cookie:
BlockClosure>>ifCurtailed:
InputState>>wndProc:message:wParam:lParam:cookie:
PushButton(ControlView)>>defaultWindowProcessing:wParam:lParam:
MouseEvent(WindowsEvent)>>defaultWindowProcessing
PushButton(View)>>onLeftButtonReleased:
PushButton(View)>>wmLButtonUp:wParam:lParam:
PushButton(View)>>dispatchMessage:wParam:lParam:
[] in InputState>>wndProc:message:wParam:lParam:cookie:
BlockClosure>>ifCurtailed:
InputState>>wndProc:message:wParam:lParam:cookie:
ShellView(View)>>isDialogMessage:
ShellView>>preTranslateMouseInput:
ShellView(View)>>preTranslateMessage:
InputState>>preTranslateMessage:
InputState>>pumpMessage:
InputState>>loopWhile:
InputState>>mainLoop
[] in InputState>>forkMain
ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry

**Comment by jgfoster** [comment]
Fixed with #90.

**Comment by ericwinger** [comment]
Confirmed fixed in #90
